charts/fission-all/values.yaml

```diff
@@ -483,6 +482,38 @@ prometheus:
 canaryDeployment:
   enabled: false
 
+## Enable authentication for fission function invocation via Fission router
+##
+authentication:
```

Comment:
Please add relevant command to retrieve username and password if authencation is enabled. 

charts/fission-all/templates/NOTES.txt

Reply:
Fixed